components/button/Button.tsx

```diff
@@ -99,6 +109,28 @@ const Button = ({
         modifier = [...modifier, ...md.topContainer];
     }
 
+    if (kind === EButtonKinds.COPY_TO_CLIPBOARD) {
+      modifier = [
+        ..._.difference(modifier, [
```

Comment:
I have never worked with Loadash so far. Do you need it here to compare the values of the arrays and only add the different ones?

Reply:
yes, it is only intended to remove the items on the second array. Instead of making a filter with different cases.